src/main/java/talkabot/Storage.java

```diff
@@ -0,0 +1,93 @@
+package talkabot;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+
+import talkabot.task.Deadline;
+import talkabot.task.Event;
+import talkabot.task.Task;
+import talkabot.task.TaskList;
+import talkabot.task.ToDo;
+
+/**
+ * Storage Class handles the file containing the task list
+ * to be read and saved onto the hard drive.
+ */
+public class Storage {
+    private String filePath;
+
+    /**
+     * Constructor for Storage class.

```

Comment:
Should this be "Constructs a Storage class..." instead? 
```suggestion
     * Constructs a Storage class...

```